### ISSUE TITLE
AidboxProfile creation fails

### ISSUE THREAD

**Issue by orlandow** [Aidbox/Issues#16]
I'm trying to create a new AidboxProfile based on the tutorial [here](https://docs.aidbox.app/api/profiling)

```
  POST /AidboxProfile
  Content-Type: application/yaml

  resourceType: AidboxProfile
  id: profile1
  bind:
    id: Patient
    resourceType: Entity
  schema:
    type: object
    required:
    - name
    - gender
```

But it fails with:

```
{
  "resourceType": "OperationOutcome",
  "issue": [
    {
      "severity": "fatal",
      "diagnostics": "org.postgresql.util.PSQLException: ERROR: relation \"metadata_version\" does not exist\n  Position: 16\n at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse (QueryExecutorImpl.java:2458)\n    org.postgresql.core.v3.QueryExecutorImpl.processResults (QueryExecutorImpl.java:2158)\n    org.postgresql.core.v3.QueryExecutorImpl.execute (QueryExecutorImpl.java:291)\n    org.postgresql.jdbc.PgStatement.executeInternal (PgStatement.java:432)\n    org.postgresql.jdbc.PgStatement.execute (PgStatement.java:358)\n    org.postgresql.jdbc.PgPreparedStatement.executeWithFlags (PgPreparedStatement.java:171)\n    org.postgresql.jdbc.PgPreparedStatement.executeQuery (PgPreparedStatement.java:119)\n    com.zaxxer.hikari.pool.ProxyPreparedStatement.executeQuery (ProxyPreparedStatement.java:52)\n    com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeQuery (HikariProxyPreparedStatement.java:-1)\n    clojure.java.jdbc$db_query_with_resultset$run_query_with_params__3517.invoke (jdbc.clj:831)\n    clojure.java.jdbc$db_query_with_resultset.invokeStatic (jdbc.clj:847)\n    clojure.java.jdbc$db_query_with_resultset.invoke (jdbc.clj:818)\n    clojure.java.jdbc$query.invokeStatic (jdbc.clj:874)\n    clojure.java.jdbc$query.invoke (jdbc.clj:854)\n    clojure.java.jdbc$query.invokeStatic (jdbc.clj:867)\n    clojure.java.jdbc$query.invoke (jdbc.clj:854)\n    clj_pg.honey$query.invokeStatic (honey.clj:118)\n    clj_pg.honey$query.invoke (honey.clj:110)\n    clojure.lang.AFn.applyToHelper (AFn.java:156)\n    clojure.lang.RestFn.applyTo (RestFn.java:132)\n    clojure.core$apply.invokeStatic (core.clj:659)\n    clojure.core$apply.invoke (core.clj:652)\n    clj_pg.honey$query_first.invokeStatic (honey.clj:145)\n    clj_pg.honey$query_first.doInvoke (honey.clj:143)\n    clojure.lang.RestFn.applyTo (RestFn.java:139)\n    clojure.core$apply.invokeStatic (core.clj:659)\n    clojure.core$apply.invoke (core.clj:652)\n    clj_pg.honey$query_value.invokeStatic (honey.clj:148)\n    clj_pg.honey$query_value.doInvoke (honey.clj:147)\n    clojure.lang.RestFn.invoke (RestFn.java:423)\n    proto.crud$commit_metadata_BANG_.invokeStatic (crud.clj:265)\n    proto.crud$commit_metadata_BANG_.invoke (crud.clj:262)\n    proto.operations$create_STAR_.invokeStatic (operations.clj:208)\n    proto.operations$create_STAR_.invoke (operations.clj:195)\n    proto.operations$eval10775$fn__10777.invoke (operations.clj:242)\n    clojure.lang.MultiFn.invoke (MultiFn.java:229)\n    unifn.core$_STAR_apply_impl$fn__209.invoke (core.clj:53)\n    unifn.core$_STAR_apply_impl.invokeStatic (core.clj:53)\n    unifn.core$_STAR_apply_impl.invoke (core.clj:35)\n    unifn.core$_STAR_apply.invokeStatic (core.clj:79)\n    unifn.core$_STAR_apply.invoke (core.clj:75)\n    proto.operations$eval10783$fn__10785.invoke (operations.clj:257)\n    clojure.lang.MultiFn.invoke (MultiFn.java:229)\n    unifn.core$_STAR_apply_impl$fn__209.invoke (core.clj:53)\n    unifn.core$_STAR_apply_impl.invokeStatic (core.clj:53)\n    unifn.core$_STAR_apply_impl.invoke (core.clj:35)\n    unifn.core$_STAR_apply.invokeStatic (core.clj:79)\n    unifn.core$_STAR_apply.invoke (core.clj:75)\n    unifn.core$_STAR_apply.invokeStatic (core.clj:84)\n    unifn.core$_STAR_apply.invoke (core.clj:75)\n    proto.operations$call_inside_transaction$fn__10756.invoke (operations.clj:186)\n    clojure.java.jdbc$db_transaction_STAR_.invokeStatic (jdbc.clj:613)\n    clojure.java.jdbc$db_transaction_STAR_.invoke (jdbc.clj:585)\n    clojure.java.jdbc$db_transaction_STAR_.invokeStatic (jdbc.clj:629)\n    clojure.java.jdbc$db_transaction_STAR_.invoke (jdbc.clj:585)\n    clojure.java.jdbc$db_transaction_STAR_.invokeStatic (jdbc.clj:598)\n    clojure.java.jdbc$db_transaction_STAR_.invoke (jdbc.clj:585)\n    proto.operations$call_inside_transaction.invokeStatic (operations.clj:184)\n    proto.operations$call_inside_transaction.invoke (operations.clj:181)\n    proto.operations$eval10791$fn__10792.invoke (operations.clj:262)\n    clojure.lang.MultiFn.invoke (MultiFn.java:229)\n    unifn.core$_STAR_apply_impl$fn__209.invoke (core.clj:53)\n    unifn.core$_STAR_apply_impl.invokeStatic (core.clj:53)\n    unifn.core$_STAR_apply_impl.invoke (core.clj:35)\n    unifn.core$_STAR_apply.invokeStatic (core.clj:79)\n    unifn.core$_STAR_apply.invoke (core.clj:75)\n    unifn.routing$eval5996$fn__5998.invoke (routing.clj:29)\n    clojure.lang.MultiFn.invoke (MultiFn.java:229)\n    unifn.core$_STAR_apply_impl$fn__209.invoke (core.clj:53)\n    unifn.core$_STAR_apply_impl.invokeStatic (core.clj:53)\n    unifn.core$_STAR_apply_impl.invoke (core.clj:35)\n    unifn.core$_STAR_apply.invokeStatic (core.clj:79)\n    unifn.core$_STAR_apply.invoke (core.clj:75)\n    unifn.core$_STAR_apply.invokeStatic (core.clj:84)\n    unifn.core$_STAR_apply.invoke (core.clj:75)\n    aidbox.box$dispatch_to_box.invokeStatic (box.clj:43)\n    aidbox.box$dispatch_to_box.invoke (box.clj:37)\n    aidbox.core$eval17952$fn__17954.invoke (core.clj:229)\n    clojure.lang.MultiFn.invoke (MultiFn.java:229)\n    unifn.core$_STAR_apply_impl$fn__209.invoke (core.clj:53)\n    unifn.core$_STAR_apply_impl.invokeStatic (core.clj:53)\n    unifn.core$_STAR_apply_impl.invoke (core.clj:35)\n    unifn.core$_STAR_apply.invokeStatic (core.clj:79)\n    unifn.core$_STAR_apply.invoke (core.clj:75)\n    unifn.core$_STAR_apply.invokeStatic (core.clj:84)\n    unifn.core$_STAR_apply.invoke (core.clj:75)\n    unifn.rest$handle.invokeStatic (rest.clj:27)\n    unifn.rest$handle.invoke (rest.clj:23)\n    unifn.rest$restart$fn__8024.invoke (rest.clj:61)\n    org.httpkit.server.HttpHandler.run (RingHandler.java:91)\n    java.util.concurrent.Executors$RunnableAdapter.call (Executors.java:511)\n    java.util.concurrent.FutureTask.run (FutureTask.java:266)\n    java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1142)\n    java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:617)\n    java.lang.Thread.run (Thread.java:745)\n",
      "code": "exception"
    }
  ]
}

```

tag: e443f2f

**Comment by Aitem** [comment]
Hi @orlandow 
This issue is fixed in `c1b1112`

**Comment by orlandow** [comment]
I'm closing this... :+1: working for me
